src/main/kotlin/no/nav/syfo/clients/Tilgangskontroll.kt

```diff
@@ -0,0 +1,60 @@
+package no.nav.syfo.clients
+
+import io.ktor.client.*
+import io.ktor.client.engine.cio.*
+import io.ktor.client.features.*
+import io.ktor.client.features.json.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
+import net.logstash.logback.argument.StructuredArguments
+import no.nav.syfo.Environment
+import no.nav.syfo.domain.Fodselsnummer
+import no.nav.syfo.util.bearerHeader
+import org.slf4j.LoggerFactory
+
+class Tilgangskontroll(env: Environment) {
```

Comment:
Trenger ikke sende inn parametre som ikke brukes.

Reply:
Trenger vel env for å kunne vite om det er dev eller prod?

Reply:
Environment er et objekt som inneholder mange parametre, såvidt jeg kan se brukes bare en av de?